backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -44,6 +45,8 @@ public void processPersonCSV(InputStream csvStream) throws IOException {
             final Map<String, String> row = valueIterator.next();
 
             final LocalDate patientDOB = LocalDate.parse(row.get("DOB"));
+            ArrayList<String> race = new ArrayList<String>();
```

Comment:
Can just be `Collections.singletonList(row.get("Race")`—but do we want to allow setting multiple race values in the CSV? Or is that for another day?

Reply:
I think it is another day